configs/relengworker-prod/config.yml

```diff
@@ -513,7 +569,7 @@ worker_types:
     autoscale:
       algorithm: slo
       args:
-        max_replicas: 1
+        max_replicas: 3
```

Comment:
https://firefox-ci-tc.services.mozilla.com/tasks/groups/IiPerdPzQ16jfNHgkLUYXQ usually has 12 tasks. Let's bump to `3` to be consistent with beetmoverworkers in delivering a release.